CONTRIBUTING.md

```diff
@@ -6,9 +6,9 @@ worst that can happen is that you'll be politely asked to change something. We
 appreciate any sort of contributions and don't want a wall of rules to get in
 the way of that.
 
-However, for those individuals who want a bit more guidance on the best way to
-contribute to the project, read on. This document will cover what we're looking
-for. By addressing all the points we're looking for, it raises the chances we
+However, for those who want a bit more guidance on the best way to
```

Comment:
This sentence is already re-factored in an earlier PR. Can you undo the changes here?